fix 'make awx-kube-build' fail when ui-next src already exist

### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Bug Summary

```
 => ERROR [builder 11/12] RUN make sdist && /var/lib/awx/venv/awx/bin/pip install dist/awx.tar.gz      4.4s
------
 > [builder 11/12] RUN make sdist && /var/lib/awx/venv/awx/bin/pip install dist/awx.tar.gz:
#18 0.481 Unable to import setuptools-scm, attempting to install now...
#18 2.738 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#18 4.229 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#18 4.366 === Setting up awx/ui_next/src ===
#18 4.391 === Updating awx/ui_next/src from main of https://github.com/ansible/ansible-ui.git ===
#18 4.395 fatal: not a git repository (or any of the parent directories): .git
#18 4.396 make: *** [awx/ui_next/Makefile:46: awx/ui_next/src] Error 128
------
executor failed running [/bin/sh -c make sdist && /var/lib/awx/venv/awx/bin/pip install dist/awx.tar.gz]: exit code: 2
make: *** [awx-kube-build] Error 1
```

### AWX version

awx: 21.13.1.dev101+g6908f415a1

### Select the relevant components

- [ ] UI
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [X] Other

### Installation method

kubernetes

### Modifications

no

### Ansible version

_No response_

### Operating system

_No response_

### Web browser

_No response_

### Steps to reproduce

`make ui-next/src`
`make awx-kube-build`

### Expected results

awx image build successfully

### Actual results

awx image build failed

### Additional information

_No response_

--- a/awx/ui_next/Makefile
+++ b/awx/ui_next/Makefile
@@ -16,9 +16,10 @@ ui-next: ui-next/build
 .PHONY: ui-next/build
 ## Build ui-next/build
 ui-next/build: $(UI_NEXT_DIR)/build
-.PHONY: $(UI_NEXT_DIR)/build
+
 ## True build target for ui-next.
-$(UI_NEXT_DIR)/build: ui-next/src/build
+$(UI_NEXT_DIR)/build:
+	@$(MAKE) $(UI_NEXT_DIR)/src/build/awx
 	@echo "=== Copying $(UI_NEXT_DIR)/src/build to $(UI_NEXT_DIR)/build ==="
 	@rm -rf $(UI_NEXT_DIR)/build
 	@cp -r $(UI_NEXT_DIR)/src/build $(UI_NEXT_DIR)
@@ -26,11 +27,10 @@ $(UI_NEXT_DIR)/build: ui-next/src/build
 
 .PHONY: ui-next/src/build
 ## Build ui-next/src/build
-ui-next/src/build: $(UI_NEXT_DIR)/src/build
+ui-next/src/build: $(UI_NEXT_DIR)/src/build/awx
 
-.PHONY: $(UI_NEXT_DIR)/src/build
-## True target for ui-next/src/build.
-$(UI_NEXT_DIR)/src/build: ui-next/src ui-next/src/webpack
+## True target for ui-next/src/build. Build ui_next from source.
+$(UI_NEXT_DIR)/src/build/awx: $(UI_NEXT_DIR)/src $(UI_NEXT_DIR)/src/node_modules/webpack
 	@echo "=== Building ui_next ==="
 	@cd $(UI_NEXT_DIR)/src && npm run build:awx
 
@@ -39,7 +39,6 @@ $(UI_NEXT_DIR)/src/build: ui-next/src ui-next/src/webpack
 ui-next/src: $(UI_NEXT_DIR)/src
 
 # TODO: Rewrite this big bash script in a more readable way.
-.PHONY: $(UI_NEXT_DIR)/src
 ## True target for ui-next/src.
 $(UI_NEXT_DIR)/src:
 	@echo "=== Setting up $(UI_NEXT_DIR)/src ==="
@@ -81,7 +80,6 @@ $(UI_NEXT_DIR)/src:
 ## Install webpack.
 ui-next/src/webpack: $(UI_NEXT_DIR)/src/node_modules/webpack
 
-.PHONY: $(UI_NEXT_DIR)/src/node_modules/webpack
 ## True target for ui-next/src/webpack.
 $(UI_NEXT_DIR)/src/node_modules/webpack:
 	@echo "=== Installing webpack ==="
@@ -100,3 +98,15 @@ clean/ui-next/src:
 ## Clean ui_next build
 clean/ui-next/build:
 	rm -rf $(UI_NEXT_DIR)/build
+
+.PHONY: $(UI_NEXT_DIR)/clean
+## Alias for clean/ui-next, so we can run `make clean` directly in ui_next
+$(UI_NEXT_DIR)/clean: clean/ui-next
+
+.PHONY: $(UI_NEXT_DIR)/clean/src
+## Alias for clean/ui-next/src, so we can run `make clean/src` directly in ui_next
+$(UI_NEXT_DIR)/clean/src: clean/ui-next/src
+
+.PHONY: $(UI_NEXT_DIR)/clean/build
+## Alias for clean/ui-next/build, so we can run `make clean/build` directly in ui_next
+$(UI_NEXT_DIR)/clean/build: clean/ui-next/build

--- a/awx/ui_next/README.md
+++ b/awx/ui_next/README.md
@@ -34,6 +34,12 @@ export UI_NEXT_LOCAL = /path/to/your/ui_next
 make ui-next
 ```
 
+## Rebuild
+
+```bash
+make -B ui-next
+```
+
 ## Clean
 
 ```bash

